src/main/resources/swagger.api/schemas/indexRequestBody.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "object",
+  "description": "Index request body",
+  "properties": {
+    "resourceName": {
+      "description": "Name of resource",
+      "type": "string"
+    }
+  }
```

Comment:
Should we added:
* `additionalProperties: false`
* `required: ["resourceName"]`?